ethereum/core/src/testFixtures/java/tech/pegasys/teku/core/signatures/TestSigner.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.core.signatures;
+
+import static tech.pegasys.teku.core.signatures.SigningRootUtil.signingRootForSignAggregateAndProof;
+import static tech.pegasys.teku.core.signatures.SigningRootUtil.signingRootForSignAggregationSlot;
+import static tech.pegasys.teku.core.signatures.SigningRootUtil.signingRootForSignAttestationData;
+import static tech.pegasys.teku.core.signatures.SigningRootUtil.signingRootForSignBlock;
+import static tech.pegasys.teku.core.signatures.SigningRootUtil.signingRootForSignVoluntaryExit;
+
+import org.apache.tuweni.bytes.Bytes;
+import tech.pegasys.teku.bls.BLS;
+import tech.pegasys.teku.bls.BLSKeyPair;
+import tech.pegasys.teku.bls.BLSSignature;
+import tech.pegasys.teku.datastructures.blocks.BeaconBlock;
+import tech.pegasys.teku.datastructures.operations.AggregateAndProof;
+import tech.pegasys.teku.datastructures.operations.AttestationData;
+import tech.pegasys.teku.datastructures.operations.VoluntaryExit;
+import tech.pegasys.teku.datastructures.state.ForkInfo;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+
+public class TestSigner implements Signer {
```

Comment:
Is this needed? It looks almost the same as the LocalSigner.

Reply:
LocalSigner uses asyncRunner. This one doesn't. May be I can modify LocalSigner slightly so that it can sign both sync and async mode. Then this can be deleted.

Reply:
Just use `LocalSigner` with `SyncAsyncRunner` like we do in all the other tests.

Reply:
Thanks for this hint @ajsutton, I was missing this bit :).